### PR TITLE
[VPU][DTS][Tests] Remove WAs for networks with multiple outputs

### PR DESCRIPTION
**Pull request by AndrewBakalinIntel**
Ticket - #41403
It has been partially done in https://github.com/openvinotoolkit/openvino/pull/2698. This PR removes WA from other tests.

### CODE REVIEW

**Comment by AndrewBakalinIntel** [comment]
@gladilov-gleb @Maxim-Doronin could you please take a look?

File: inference-engine/tests/functional/plugin/myriad/ngraph/transformations/dynamic_to_static_shape_split.cpp [hunk 1]
```diff
@@ -65,12 +65,9 @@ std::shared_ptr<const ngraph::Function> transform(
     const auto dsr = std::make_shared<ngraph::vpu::op::DynamicShapeResolver>(data, dims);
     const auto node = std::make_shared<ngraph::opset5::Split>(dsr, axis, splitSetup.numSplits);
 
-    // tests are capable to compare functions with one result only
-    const auto testsWa = std::make_shared<ngraph::opset5::Concat>(node->outputs(), splitSetup.axis);
-
     auto outputShape = node->get_output_partial_shape(0);
     const auto function = std::make_shared<ngraph::Function>(
-            ngraph::NodeVector{testsWa},
+            ngraph::OutputVector{node->outputs()},
```
Comment (gladilov-gleb) [inline inference-engine/tests/functional/plugin/myriad/ngraph/transformations/dynamic_to_static_shape_split.cpp:70]: Does `ngraph::Node::outputs` return `ngraph::OutputVector` already?

Comment (AndrewBakalinIntel) [reply inference-engine/tests/functional/plugin/myriad/ngraph/transformations/dynamic_to_static_shape_split.cpp]: Yes, good catch, fixed.

**Review by Maxim-Doronin — APPROVED** [review]

**Review by gladilov-gleb — APPROVED** [review]

**Comment by AndrewBakalinIntel** [comment]
@Maxim-Doronin could you please merge?

**Pull request merged**